### PR TITLE
{backend, test}: add initial backend tests

### PR DESCRIPTION
**Pull request by acharb**
**WHAT**
Add testing to the dashboard backend.

**WHY**
Testing will help prevent future bugs while this app is getting refactored. Adding the packages and a simple test at first but more test coverage will probably be added/needed in the future.

Also moved the interval calls of `updateApiLumens`, which updates the cache, to be called in app.js. This is so when importing the functions to be tested the functions are not invoked, as it is currently doing.

### CODE REVIEW

**Comment by acharb** [comment]
Not sure what these Netlify checks are, but they aren't blocking merge, and Ops is looking into them
https://stellarfoundation.slack.com/archives/C02U19A2A/p1638465561444900 cc @tprobinson

File: package.json
Comment (leighmcculloch) [inline package.json:56]: ❓ I'm not very familiar with how dependencies work, so I could be way off the mark with this, but since mocha is a test framework, should it be in the `devDependencies`?

File: test/unit/backend.js
Comment (leighmcculloch) [inline test/unit/backend.js:30]: ❓ Are there other things we can test these functions on, like some side-effect they have?

**Review by leighmcculloch — APPROVED** [review]
I left two questions, but neither blocking and it looks good to me.

File: test/unit/backend.js
Comment (acharb) [reply test/unit/backend.js:30]: yup! I plan on adding tests for those. Didn't want to make this PR too big

File: package.json
Comment (acharb) [reply package.json]: ya  I wasn't sure if the github action environment would install devDependencies, but looks like it does 👍  moved
https://github.com/stellar/dashboard/pull/188/commits/cc7de972a00f5134ffdd4bc92cca6ceffcfb27c4

**Comment by acharb** [comment]
@leighmcculloch can you approve again? I'll change the dismiss stale review flag in terraform

**Review by leighmcculloch — APPROVED** [review]
🎉

**Pull request merged**
